src/lib/components/pages/anime_info/index.svelte

```diff
@@ -761,13 +791,21 @@
 </div>
 
 <style lang="scss">
-    episode-japanese-name,
-    episode-name {
-        &:not(.group-hover) {
-            /* if we need to change the width, we should change the 90% to higher  */
-            mask-image: linear-gradient(90deg, rgba(7, 5, 25, 0.95) 90%, rgba(0, 0, 0, 0) 100%);
-            mask-repeat: no-repeat;
-            mask-position: right;
+    // tailwind trasnitions seems not working
+    @media (min-width: 768px) {
+        episode-info-card {
+            transition: max-height 0.2s ease-in-out;
+
+            scroll-area-title {
+                transition: max-height 0.2s ease-in-out;
+                overflow-y: hidden;
+                height: auto;
+                max-height: 1vw;
+
+                &:hover {
+                    max-height: var(--max-height-hover, 3.75vw);
+                }
+            }
         }
     }
```

Comment:
We should convert it to tailwind classes :)

want me to work on this?

Reply:
>We should convert it to tailwind classes :)
want me to work on this?

Yea duh, but you want to refactor js?